db/seeds.rb

```diff
@@ -8,4 +8,17 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
-User.create email: "test@test.com", password: "password"
+case Rails.env
+when "development", "test"
+  seed_development
```

Comment:
I think the method needs to be defined before that point, as it's just evaluated from top to bottom i think